src/main/java/acme/entities/lectures/Lecture.java

```diff
@@ -0,0 +1,46 @@
+
+package acme.entities.lectures;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+
+import org.hibernate.validator.constraints.Length;
+import org.hibernate.validator.constraints.URL;
+
+import acme.entities.enumerates.Type;
+import acme.framework.data.AbstractEntity;
+
```

Comment:
Faltan anotaciones Getter, Setter, Entity
```suggestion
@Getter 
@Setter
@Entity
```